File: leetcode/Accepted/shuffle-string.py
class Solution:
    def restoreString(self, s: str, indices: List[int]) -> str:
        result = [0]*len(s)
        i = 0
        for index in indices:
            result[index] = s[i]
            i += 1

        string = ''
        return (string.join(result))

File: leetcode/Accepted/shuffle-string.cpp
class Solution {
  public:
    string restoreString(string s, vector<int>& indices) {
      string result = s;
      for(int i = 0; i < (int) s.size(); ++i){
        result[indices[i]] = s[i];
      }
      return result;
    }
};
